tck/src/main/java/org/eclipse/configjsr/CDIPropertyNameMatchingTest.java

```diff
@@ -119,7 +127,7 @@ public void can_inject_simple_values_when_defined() {
         @Inject
         @ConfigProperty(name="my.int.property")
         private Provider<Integer> intProperty;
```

Comment:
Can you remove the Provider so the injection is static, as this test is not to test dynamic behavior? In this way, the assertThat does not need to do a get()?

Reply:
done